integration/nevermined/Subscriptions.e2e.test.ts

```diff
@@ -202,11 +203,15 @@ describe('Subscriptions using NFT ERC-721 End-to-End', () => {
             )
             assert.isTrue(receipt)
 
+            const tokenId = utils.keccak256(
+                utils.defaultAbiCoder.encode(
+                    ['bytes32', 'bytes32'],
+                    [zeroX(subscriptionDDO.shortId()), agreementId]
+                )
+            )
+
```

Comment:
I think `nfts.ownerOf` should have two implementations and the tokenId logic should be done there:

```ts
public async ownerOf(tokenId: string): string
public async ownerOf(did: string, agreementId: string): string
```